utils/zerotrie/src/zerotrie.rs

```diff
@@ -234,6 +261,15 @@ pub struct ZeroTrieExtendedCapacity<Store: ?Sized> {
     pub store: Store,
 }
 
+impl<Store: ?Sized> ZeroTrieExtendedCapacity<Store> {
+    fn transparent_ref_from_store(s: &Store) -> &Self {
```

Comment:
Thought: this is a new naming convention. I think in some places we use `ref_cast`

Reply:
It's an internal API, and I wanted it to be abundantly clear what it was doing